assets/dev/js/editor/document/ui-states/scrubbing-mode.js

```diff
@@ -0,0 +1,22 @@
+import UiStateBase from 'elementor-api/core/states/ui-state-base';
```

Comment:
```suggestion
// TODO: Move the `editor` component when it's available, because it's not a document related state.
import UiStateBase from 'elementor-api/core/states/ui-state-base';
```

Reply:
Done.